R/utils_formatters.R

```diff
@@ -256,7 +256,10 @@ format_num_to_str <- function(x,
                               drop_trailing_zeros,
                               drop_trailing_dec_mark,
                               format = "f",
-                              replace_minus_mark = TRUE) {
+                              replace_minus_mark = TRUE,
+                              system = c("intl", "ind")) {
+
```

Comment:
If `system` is `ind`, hard code `sep_mark` and `dec_mark` here. (Maybe warn if they're not missing and not the hardcoded values?)